aspnet/publishing/iis.rst

```diff
@@ -1,116 +1,187 @@
 Publishing to IIS
-=============================
+=================
 
-By `Rick Anderson`_ and `Luke Latham`_
+By `Rick Anderson`_ and `Luke Latham <https://github.com/GuardRex>`_
 
-.. contents:: Sections:
-  :local:
-  :depth: 1
+- `Supported Operating Systems`_
+- `IIS Configuration`_
+- `Install the Microsoft ASP.NET Core Module`_
+- `Install .NET Core on the Server`_
+- `Deploy the Application`_
+- `Configure the Website in IIS`_
+- `Create a Data Protection Registry Hive`_
+- `Common Errors`_
+- `Additional Resources`_
 
-Install the HTTP Platform Handler
------------------------------------
+Supported Operating Systems
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-- Install the HTTP Platform Handler version 1.2 or higher:
+The following operating systems are supported:
 
-  - `64 bit HTTP Platform Handler <http://go.microsoft.com/fwlink/?LinkID=690721>`_
-  - `32 bit HTTP Platform Handler <http://go.microsoft.com/fwlink/?LinkId=690722>`_
+- Windows 7 and newer
+- Windows Server 2008 R2 and newer\*
 
-If you need to enable IIS, see `IIS server configuration`_.
+\*Conceptually, the IIS configuration described in this document also applies to hosting ASP.NET Core applications on Nano Server IIS, but refer to `ASP.NET Core on Nano Server <https://docs.asp.net/en/latest/tutorials/nano-server.html>`_ for specific instructions.
```

Comment:
Two questions ...
1. That should be \`blah <http://foo>\`__ ... correct?
2. all inline hyperlinks? http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#anonymous-hyperlinks ... I'm confused. The descriptions on how and why they exist seems at odds with doing it everywhere. Just to be clear: You want **all** inline hyperlinks with the double-underscore? 

btw- That aspect of links is not in the style documentation.